src/js/player.js

```diff
@@ -3444,6 +3444,16 @@ class Player extends Component {
     this.on('mousemove', handleMouseMove);
     this.on('mouseup', handleMouseUp);
 
+    this.on(this.getChild('controlBar'), 'mouseenter', function(event) {
+      this.options_.inactivityTimeout = 0;
+
+    });
+
+    this.on(this.getChild('controlBar'), 'mouseleave', function(event) {
+      this.options_.inactivityTimeout = 2000;
```

Comment:
if someone set a custom inactivity timeout, this will reset that value.

Reply:
We could store the inactivityTimeout in `this.cache_` and then reset to that value.

Reply:
Good point, I now save the value in the cache object on mouseenter and retrieve it on mouseleave